src/general/StringUtils.cs

```diff
@@ -187,4 +187,37 @@ public static string Unescape(this string input)
 
         return result.ToString();
     }
+
+    /// <summary>
+    ///   Convert a value in a string with up to 3 digits.
+    /// </summary>
+    /// <remarks>
+    ///   If greater than 100, it should use F0, otherwise it should use F1.
+    ///   Above 1000 it should use the number formatting which uses K for
+    ///   thousands, M for millions etc.
+    /// </remarks>
+    /// <param name="value">The value to format</param>
+    /// <returns>The formatted string</returns>
+    public static string ThreeDigitFormat(double value)
+    {
+        if (value >= 1000)
+            return FormatNumber(value);
+
+        if (value >= 100)
+            return value.ToString("F0", CultureInfo.CurrentCulture);
+
+        return value.ToString("F1", CultureInfo.CurrentCulture);
+    }
+
+    /// <summary>
+    ///   Formats two numbers separated by a slash. The numbers will have
+    ///   up to 3 digits each.
+    /// </summary>
+    /// <param name="v1">The first number (numerator)</param>
```

Comment:
`v` is not an approved single letter variable, please rename the v1 and v2 variables to something else, for example "value1" and "value2".

Reply:
I went with 'numerator' and 'denominator' here since this is a fraction and seems slightly more descriptive.